lib/myFTClient.js

```diff
@@ -676,6 +718,17 @@ function getGroupsFollowingConcept(licenceId, conceptId) {
 }
 
 /**
+ * Gets the users that are following a concept/topic as an individual
+ * @param {String} licenceId -
+ * @param {String} conceptId -
+ * @returns {Promise} response -
+ */
+function getUsersFollowingConceptAsIndividual(licenceId, conceptId) {
```

Comment:
And we're also going to need a method to extract group follows (with the new `followed_by_kat` relationship)